components/atoms/sections/Card/card.module.css

```diff
@@ -4,6 +4,11 @@
   border-radius: var(--spacing-em-tiny);
 }
 
+.disabled {
+  opacity: 0.5;
+  cursor: not-allowed;
```

Comment:
this property would be better applied on the wrapping clickable element (`button`, `a`)